lib/unicode_plot.rb

```diff
@@ -1,3 +1,5 @@
+require 'stringio'
```

Comment:
Could you please put this line at the beginning of `lib/unicode_plot/styled_printer.rb` instead of here?

Reply:
`StringIO` is used not only in `styled_printer.rb`, but also `plot.rb`.
So I think this line should remain here.

I'm sorry about my wrong instruction. Please ignore the previous comment.

Reply:
Okay. I have pushed again. I left this require as-is since it would otherwise need to go into two different files. Let me know if this is acceptable or if you require further changes.